### DISCUSSION TITLE
Where or how are the Regex patterns sourced

### DISCUSSION THREAD

**Discussion by realashleybailey** [ThioJoe/YT-Spammer-Purge#778] [General]
These regular expressions are obviously not hand made, how are they generated or sourced, is there a way to test them accurately

**Comment by ThioJoe** [comment]
They're generated with https://pypi.org/project/confusables/

But the program actually uses the customConfusables folder where I've added additional mappings for that module

**Reply by realashleybailey** [reply]
Thank you Joe
